torch/multiprocessing/spawn.py

```diff
@@ -205,12 +206,13 @@ def __init__(self, processes, error_files):
 # Currently we only add this API first, we can consider adding it to documentation as
 # needed in the future.
 def start_processes(
-    fn, args=(), nprocs=1, join=True, daemon=False, start_method="spawn"
+    fn, args=(), nprocs=1, join=True, daemon=False, start_method="spawn", start_func_in_parallel=False
 ):
+    assert not start_func_in_parallel or start_method != "fork", "fork is not supported for start_func_in_parallel"
```

Comment:
Since this is a performance optimization can we instead warn and use the non-threadpool path?

Reply:
changed to fallback to sequential if not forkserver or env in next version